app/pages/analyst/admin/download-attachments.tsx

```diff
@@ -1,37 +1,125 @@
+import { ChangeEvent, useState } from 'react';
 import { usePreloadedQuery } from 'react-relay/hooks';
 import { withRelay, RelayProps } from 'relay-nextjs';
 import { graphql } from 'react-relay';
-import { DashboardTabs } from 'components/AnalystDashboard';
 import styled from 'styled-components';
+import { DateTime } from 'luxon';
 import defaultRelayOptions from 'lib/relay/withRelayOptions';
-import { Layout } from 'components';
+import { DashboardTabs } from 'components/AnalystDashboard';
+import { ButtonLink, Layout } from 'components';
 import AdminTabs from 'components/Admin/AdminTabs';
 import { downloadAttachmentsQuery } from '__generated__/downloadAttachmentsQuery.graphql';
 
 const getDownloadAttachmentsQuery = graphql`
   query downloadAttachmentsQuery {
     session {
       sub
-      ...DashboardTabs_query
+      ...DashboardTabs_query  
+    }
+    allIntakes {
```

Comment:
```suggestion
    allIntakes(orderBy: CLOSE_TIMESTAMP_DESC) {
```

I think you could simplify this and sort intakes using graphql and remove `sortedNodes` if you wanted to!

Reply:
Done